docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -166,7 +166,7 @@ Now, select **Yes** to enable SPI:
 Finally, select **Finish**.
 Restart your Pi using `sudo reboot` to make these changes take effect.
 
-### Test your Soil Moisture readings on your Pi
+### Test your soil moisture readings on your Pi
```

Comment:
This whole section should be gone, as the modular resource abstracts this away

Reply:
i will rework this section in a different pr, i think there are some sections that could stay, like update/upgrading the pi, creating a plant-watering directory for the code, etc.